src/software/thunderscope/thunderscope_config.py

```diff
@@ -289,6 +313,13 @@ def configure_two_ai_gamecontroller_view(
     # Must be called before widgets are initialized below
     initialize_application()
 
+    # setup frametime counter
+    blue_refresh_func_frametime_counter = FrameTimeCounter()
+    blue_buffer_func_frametime_counter = FrameTimeCounter()
```

Comment:
what is the buffer_func_frametime_counter tracking?

Reply:
Or I guess I'm confused why we have two counters for the refresh and buffer swap. what does one metric give us that another one doesnt?